administrator/components/com_templates/tmpl/template/default.php

```diff
@@ -85,23 +84,16 @@
 				<?php if (!empty($this->source->coreFile)) : ?>
 					<?php $coreFileContent = file_get_contents($this->source->coreFile); ?>
 					<?php $overrideFileContent = file_get_contents(JPATH_SITE . '/templates/' . $this->template->element . $this->source->filename); ?>
-
-					<table id="diff" class="table">
-						<thead>
-							<tr>
-								<th style="display:none"><?php echo JText::_('COM_TEMPLATES_COMPARE_ORIGINAL'); ?></th>
-								<th style="display:none"><?php echo JText::_('COM_TEMPLATES_COMPARE_CHANGED'); ?></th>
-								<th><?php echo JText::sprintf('COM_TEMPLATES_COMPARE_DIFF') . ' ' .$this->source->coreFile; ?></th>
-							</tr>
-						</thead>
-						<tbody>
-						<tr>
-							<td style="display:none" class="original"><?php echo htmlspecialchars($coreFileContent, ENT_COMPAT, 'UTF-8'); ?></td>
-							<td style="display:none" class="changed" ><?php echo htmlspecialchars($overrideFileContent, ENT_COMPAT, 'UTF-8'); ?></td>
-							<td class="diff">&nbsp;</td>
-						</tr>
-						</tbody>
-					</table>
+					<div id="diff-main">
+						<p class="lead" style="display:none"><?php echo JText::_('COM_TEMPLATES_COMPARE_ORIGINAL'); ?></p>
```

Comment:
Up to know we do not need line 88 and line 89, right. You just prepared it for Allon's design in glip.

Reply:
Yes

Reply:
If we do not need it, then I would take it away and add when it is needed.